components/widgets/modals/full_screen_modal.tsx

```diff
@@ -47,9 +65,19 @@ export default class FullScreenModal extends React.Component<Props> {
                 appear={true}
             >
                 <div className='FullScreenModal'>
+                    {this.props.onGoBack &&
+                        <BackIcon
+                            tabIndex='0'
+                            className='back'
+                            onKeyDown={this.onBackKeyDown}
+                            id='backIcon'
+                            onClick={this.props.onGoBack}
+                        />}
                     <CloseIcon
+                        tabIndex='0'
                         className='close-x'
                         onClick={this.close}
+                        onKeyDown={this.onCloseKeyDown}
                         id='closeIcon'
                     />
```

Comment:
@jespino, I think this would be a perfect opportunity to make these buttons fully semantic and accessible. If we are going to the effort of making the close and back `<span>` tags keyboard navigable, we should wrap them in semantic `<button>` tags and add appropriate labels to them (title and aria-labels attributes). Thoughts?

Reply:
I'll do it

Reply:
Done